pkg/kv/kvserver/rangefeed/scheduled_processor.go

```diff
@@ -753,7 +755,7 @@ func (p *ScheduledProcessor) publishDeleteRange(
 		Span:      span,
 		Timestamp: timestamp,
 	})
-	p.reg.PublishToOverlapping(ctx, span, &event, alloc)
+	p.reg.PublishToOverlapping(ctx, span, &event, false /* omitInRangefeeds */, alloc)
```

Comment:
might be nicer to add an enum/constant to avoid passing these "false /* ... */" things....